test/net/sourceforge/kolmafia/textui/command/EatCommandTest.java

```diff
@@ -0,0 +1,48 @@
+package net.sourceforge.kolmafia.textui.command;
+
+import static internal.helpers.HttpClientWrapper.getRequests;
+import static internal.helpers.HttpClientWrapper.setupFakeClient;
+import static internal.helpers.Networking.assertPostRequest;
+import static internal.helpers.Player.withItem;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.hasSize;
+
+import internal.helpers.Cleanups;
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.objectpool.ItemPool;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class EatCommandTest extends AbstractCommandTestBase {
+  @BeforeEach
+  void beforeEach() {
```

Comment:
There's only one test so doesn't matter at the moment, but I think this should be able to be `BeforeAll` (with cleanups).